invenio_circulation/schemas/circulation/loan-v1.0.0.json

```diff
@@ -0,0 +1,79 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "title": "Schema for loan",
+  "type": "object",
+  "required": [
+    "$schema",
+    "loanid",
+    "state"
+  ],
+  "additionalProperties": false,
+  "properties": {
+    "$schema": {
+      "title": "Schema",
+      "description": "Schema to validate loans.",
+      "type": "string",
+      "minLength": 7
+    },
+    "loanid": {
```

Comment:
Is this the PID or the UUID? If the first, I would go for `loan_pid` as for the others

Reply:
Done, we are using now loan_pid everywhere.